container_files/assets/www/doc/builtin/functions/EM.md

```diff
@@ -0,0 +1,40 @@
+# EM Function
+
+The Estimation-Maximisation (EM) function type takes the output dataset of the EM procedure and applies
+it to new data in order to assign it to clusters.
+
+## Configuration
+
+![](%%config function em)
+
+Functions of this type load their internal state from a dataset, which is identified
```

Comment:
This description does not match the function parameters.  There is only one parameter dataFileUrl.